### ISSUE TITLE
Storage codes for different types of chemicals

### ISSUE THREAD

**Issue by canismarko** [canismarko/professor_oak#77]
I had a chance to look at Stanford's chemical inventory at SSRL and they have an additional field for storage category (general, flammable, oxidizer, etc.) and the secondary containers are labeled. Is this something we want to consider borrowing? Maybe good for a group meeting discussion.

**Comment by michaelplews** [comment]
You mean as an indicator of what can be stored with what? Compatibility of sorts?

**Comment by canismarko** [comment]
Yeah pretty much: http://www-ssrl.slac.stanford.edu/splab/chemicals

**Comment by michaelplews** [comment]
I like it. As long as it's not too complicated for [our users](http://i.imgur.com/tMOheW9.gif) to understand.
